extensions/azure/data-plane/storage/src/main/java/org/eclipse/dataspaceconnector/azure/dataplane/azurestorage/pipeline/AzureStorageDataSinkFactory.java

```diff
@@ -67,7 +68,7 @@ public boolean canHandle(DataFlowRequest request) {
         try {
             validateAccountName(properties.remove(AzureBlobStoreSchema.ACCOUNT_NAME));
             validateContainerName(properties.remove(AzureBlobStoreSchema.CONTAINER_NAME));
-            properties.remove(DataAddress.KEY_NAME);
+            validateKeyName(properties.remove(DataAddress.KEY_NAME));
             properties.keySet().stream().filter(k -> !DataAddress.TYPE.equals(k)).findFirst().ifPresent(k -> {
                 throw new IllegalArgumentException(format("Unexpected property %s", k));
             });
```

Comment:
Does it make sense to reuse code from `extensions/azure/data-plane/data-factory/src/main/java/org/eclipse/dataspaceconnector/azure/dataplane/azuredatafactory/AzureDataFactoryTransferRequestValidator.java` by creating one helper function?

Reply:
It usages now common code from extensions/azure/blobstorage/blob-core